soc/xtensa/intel_adsp/ace/include/intel_ace15_mtpm/adsp_shim.h

```diff
@@ -6,50 +6,68 @@
 #ifndef ZEPHYR_SOC_INTEL_ADSP_SHIM_H_
 #define ZEPHYR_SOC_INTEL_ADSP_SHIM_H_
 
-/* The "shim" block contains most of the general system control
- * registers on cAVS platforms.  While the base address changes, it
- * has remained largely, but not perfectly, compatible between
- * versions.
+#ifndef _ASMLANGUAGE
+
+/**
+ * DfPMCCU
+ * Power Management / Clock Control (HST) Registers
+ *
+ * Offset: 0x00071AC0h + 4000000h * f //f = 0 - HFC - 1
+ * These registers block (HST domain) are for general power management
+ * and clock control operation for DSP FW.
  */
+struct ace_dfpmcch {
+	uint32_t dfspsreq;	/* Offset: 0xc0 */
+	uint32_t _unused0[3];
+	uint32_t dfspsrsp;	/* Offset: 0xd0 */
+	uint32_t _unused1[1];
+	uint32_t svcfg;		/* Offset: 0xd8 */
+	uint32_t dfltrc;	/* Offset: 0xdc */
+	uint32_t _unused2[8];
+};
 
-#ifndef _ASMLANGUAGE
-struct cavs_shim {
```

Comment:
why is this being  removed?

Reply:
cavs_shim struct is useless in the ace platform. It's just a cleanup for ace...

Reply:
then please do that in a separate commit, and cleanup all the way, you are still referencing this struct in the code below.

Reply:
@nashif please re-review and resolve the conversation if it is o.k for you ;)